jobs/e2e_node/crio/templates/base/crun-enabled.yaml

```diff
@@ -0,0 +1,7 @@
+---
+storage:
+  files:
+    - path: /etc/crio/crio.conf.d/99-crun.conf
```

Comment:
I think the note is still relevant, would you mind adding it back?

Reply:
@haircommander done!